src/lib/components/Dialog.react.js

```diff
@@ -68,16 +55,10 @@ const defaultProps = {
   className: '',
   open: false,
   setProps: () => {},
-  actionsContainerClassName: null,
-  bodyClassName: null,
-  contentClassName: null,
-  overlayClassName: null,
-  paperClassName: null,
-  titleClassName: null,
   autoScrollBodyContent: false,
   useBrowserSideClose: false,
   style: null,
-  scroll: null,
+  scroll: '',
```

Comment:
This is better now that the user can change it, but let's keep the value you had from before as the default.

Reply:
When I leave it as null, I get an error. That's why I changed it

Reply:
Gotcha. Can we change it to "body" as the default instead?

Reply:
Sure